terraform/20-app/eventbridge.tf

```diff
@@ -1,30 +0,0 @@
-module "eventbridge" {
```

Comment:
This will come back post migration, but for now I don't need or want this to be triggering